install_config/install/advanced_install.adoc

```diff
@@ -223,6 +223,9 @@ All].
 validity for etcd CA, peer, server and client certificates. Defaults to `1825`
 (5 years).
 
+|`os_firewall_use_firewalld`
+|Set to `true` to use firewalld instead of the default iptables. Not available on RHEL Atomic Host. See the xref:advanced-install-configuring-firewalls[Configuring the Firewall] section for more information.
```

Comment:
This will change soon, they plan to add it to Atomic Host.  Is there a way to track this so we re-check next time AH releases?